### ISSUE TITLE
plugin documentation 

### ISSUE THREAD

**Issue by gotys** [videojs/video.js#2699]
Congratulations on using ES6 and other future-techs . However, for someone who does not have time to learn it, it is nearly IMPOSSIBLE to customize your player . I've spent 2 days trying to figure out how to write my own plugins, but unfortunatelly I am not getting anywhere, simply because you dont' have any DOCUMENTATION on how to write them. 

So I tried to learn from existing plugins but again - I am unable to install them, because there is no simple documentation on how to do so.

In short: It would be nice to have some document describing how to use Bower, Grunt..etc to even begin to use video.js plugins. Not everyone has time to learn new techs.

**Comment by misteroneill** [comment]
There is [a guide to writing and using plugins](https://github.com/videojs/video.js/blob/master/docs/guides/plugins.md) that is still correct.

You don't have to use ES6 to use video.js or to write plugins (though we recommend it!); it has changed very little in common usage from version 4.x to 5.0.

**Comment by gotys** [comment]
Can you please tell me how to get a handle on anything, for example seekBar  in plain javascript ?

For example, I've been fighting this line for 20 hours now: 

consol.log( this.controlBar.progressControl.seekBar.calculateDistance()) )

I cannot find in any docs on how to actually use the API in plain javascript :(

**Comment by gotys** [comment]
Please delete this issue,  I will write a tutorial did the test of us and post it here once ready. Thanks for your time

**Comment by mmcc** [comment]
A tutorial would be amazing @gotys, thanks! Looking forward to it.
